docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/integrate-algolia.md

```diff
@@ -708,21 +704,16 @@ class SynchronizationDependencyProvider extends SprykerSynchronizationDependency
 }
 ```
 
-### Receive ACP messages
```

Comment:
Is this section moved to another place or is it simply not needed anymore?

Reply:
it's not relevant to this integration guide, it's generic ACP functionality that's provide by default and does not require a developer action.